gax-java/gax-httpjson/src/test/java/com/google/api/gax/httpjson/RetryingTest.java

```diff
@@ -220,6 +236,9 @@ void retryWithinMaxAttempts() {
         HttpJsonCallableFactory.createUnaryCallable(
             callInt, callSettings, httpJsonCallSettings, clientContext);
     assertThat(callable.call(initialRequest)).isEqualTo(2);
+    assertThat(tracerFactory.getTracerAttempts().get()).isEqualTo(3);
+    assertThat(tracerFactory.getTracerAttemptsFailed().get()).isEqualTo(2);
+    assertThat(tracerFactory.getTracerOperationFailed().get()).isEqualTo(false);
```

Comment:
nit: Perhaps we can add TracerFailedRetriesExhausted == false check?

Reply:
Done